example/posts/views.py

```diff
@@ -1,15 +1,26 @@
-from django.shortcuts import get_object_or_404, render
+from asgiref.sync import sync_to_async
+from django.shortcuts import get_object_or_404
+
+from suspense.shortcuts import render
+from suspense.views import SuspenseTemplateView
 
 from .models import Post
 
 
-def posts(request):
+async def posts(request):
     posts = Post.objects.all()
 
     return render(request, 'posts/all.html', {'posts': posts})
 
 
-def post(request, slug):
-    post = get_object_or_404(Post, slug=slug)
+async def post(request, slug):
+    post = sync_to_async(get_object_or_404)(Post, slug=slug)
```

Comment:
At this point, it is better to add `await` to have the content displayed on the view page.

Reply:
i should rollback this part since it does not use suspense at all.

maybe i should rename the new `render` for `streaming_render` to be able to easily import and differentiate from the one from django. what do you think?

Reply:
I don't think we should change the name. `render` is fine because it is a replacement for the default django render function. If a developer uses two functions in their code at the same time, code must be like this:
```py 
from django.shortcuts import render
from suspense.shortcuts import render as suspense_render
```

Indeed for the individual item view page there is no need to use suspense. In the above example code block is a coroutine: 
```py
post = sync_to_async(get_object_or_404)(post, slug=slug)
```
It won't work in the current implementation for now. However, it seems possible to make asynchronous objects execute in the future.